src/atomizer.js

```diff
@@ -500,28 +544,74 @@ Atomizer.prototype.getCss = function (classNames/*:string[]*/, config/*:Atomizer
                 // add the dot for the class
                 className = ['.', className].join('');
 
+                // fix the comma problem in Absurd
+                // @TODO temporary until we replace Absurd
+                // See also the return of this method.
+                className = className.replace(',', '__COMMA__');
+
                 // finaly, create the object
-                csso[className] = {};
-                if (breakPoint) {
-                    csso[className][breakPoint] = {};
-                }
 
-                if (treeo.declaration) {
+                // helper rules doesn't have the same format as patterns
+                if (rule.type === 'helper') {
+                    cssoHelpers[className] = {};
+
+                    if (breakPoint) {
+                        cssoHelpers[className][breakPoint] = {};
+                    }
+                    if (!rule.declaration) {
+                        throw new Error('Declaration key is expected in a helper class. Helper class: ' + rule.prefix);
+                    }
+
                     if (breakPoint) {
-                        csso[className][breakPoint] = treeo.declaration;
+                        cssoHelpers[className][breakPoint] = rule.declaration;
                     } else {
-                        csso[className] = treeo.declaration;
+                        cssoHelpers[className] = rule.declaration;
                     }
-                } else {
-                    rule.properties.forEach(function (property) {
-                        var value = Atomizer.replaceConstants(treeo.value, options.rtl);
-                        property = Atomizer.replaceConstants(property, options.rtl);
+
+                    // we have params in declaration
+                    if (treeo.params) {
+                        treeo.params.forEach(function (param, index) {
+                            if (breakPoint) {
+                                for (var prop in cssoHelpers[className][breakPoint]) {
+                                    cssoHelpers[className][breakPoint][prop] = cssoHelpers[className][breakPoint][prop].replace('$' + index, param);
```

Comment:
I was thinking about this one. Don't you think it's better to do this in the end on top of `content` before returning it? Because I feel we're calling this method everywhere, for Helpers specifically, I'll have to loop every key in `declarations` which I already don't like to do with params right now.